src/app/event-list/event-list.component.ts

```diff
@@ -10,13 +10,20 @@ import { EventService } from '../service/event-service.service';
 export class EventListComponent implements OnInit {
 
   events: Event[];
+  alphabetizedByName: Event[];
 
   constructor(private eventService: EventService) { }
 
   ngOnInit() {
     this.eventService.findAll().subscribe(data => {
       this.events = data;
+      this.alphabetizedByName = this.events.sort(function(a, b) {return a.name.localeCompare(b.name)});
     });
   }
 
+  // alphabetizeEvents(eventsArr: Event[]) {
```

Comment:
Do we need this commented code?

Reply:
No we don't! I thought I had deleted that function completely at the bottom, but it looks like I missed the first line of it! Will delete!

Reply:
Or the whole thing... whoops!